### PR TITLE
Added new post on Introduction to GraphQL

### PR DESCRIPTION
**Pull request by pranshukharkwal**
## Type of PR

- [x] Blog/Article
- [ ] Feature/Enhancement
- [ ] Bug Fix

## Description

Wrote a blog article about GraphQL with the title , introduction to graphql | the future of apis?

## Checklist

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] If you have multiple commits please combine them into one commit by squashing them.

- [x] Read and understood the contribution guidelines at `devbae-blog/.github/CONTRIBUTING.md`.

### CODE REVIEW

**Comment by aashish-ak** [comment]
Hey @pranshukharkwal , thanks for the awesome blog, I've made a few changes to it, please review them before we merge.

**Comment by pranshukharkwal** [comment]
@aashish-ak Looks fine

**Comment by aashish-ak** [comment]
@pranshukharkwal  I wanted to point a few things which will be helpful to you in writing blogs in future:
1. Try using more pictures to describe what you're trying to say (no one likes only text in blogs).
2. Do proper indentation and spacing and try adding an index if you have more than 2 main topics.
3. Don't add link to the entire line, assign the link to a maximum of two words.

**Comment by aashish-ak** [comment]
Also don't forget to follow these [instructions](https://github.com/devbae/devbae-blog/blob/master/.github/CONTRIBUTING.md#after-your-pull-request-is-merged) after the merge :)

**Comment by pranshukharkwal** [comment]
@aashish-ak Sure thing :)

File: _drafts/2020-06-10-introduction-to-graphql.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+---
+
+If you are a developer, you would be knowing what an API is. At some point in your career, you would have used APIs to get data from some other application. Because let's be honest, [web scraping]({% post_url 2020-05-09-web-scraping %}) doesn't work all the time, plus using APIs is also a lot more convenient.
+But if API is still an alien term for you, then here is what an API means:
+
+    API stands for Application Programming Interface which is way
+    of communication between two different applications.
+    In simpler terms, API is a way of sending data from one application (server) to another application (client)
+And, when we talk of APIs, the first thing which comes to mind is REST API and the reason behind it is that immensely popular.
+Now we are not going to talk about REST API's in this article, but it is important that you need to know about them if you don't already. So here is a [simple guide](https://www.smashingmagazine.com/2018/01/understanding-using-rest-api/) to teach you all you need to know about API, REST, requests, endpoints, etc.
```
Comment (manavmehta) [inline _drafts/2020-06-10-introduction-to-graphql.md:18]: @aashish-ak this part is not readable upon deployment

Comment (aashish-ak) [reply _drafts/2020-06-10-introduction-to-graphql.md:18]: Thanks @Manav for pointing out, will fix it.

Comment (manavmehta) [reply _drafts/2020-06-10-introduction-to-graphql.md:18]: Btw, you tagged someone else.
its @manavmehta
